types/auth0-js/index.d.ts

```diff
@@ -482,7 +482,7 @@ export class CrossOriginAuthentication {
     callback(): void;
 }
 
-export type Auth0Callback<T> = (error: null | Auth0Error, result: T) => void;
+export type Auth0Callback<T, E = Auth0Error> = (error: null | E, result: T) => void;
```

Comment:
this requires the 2.3 version (as mentioned in failing CI tests):
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-3.html
Can we add the 2.3 requiment pragma(s) as here?
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/29101

Reply:
Thank you @peterblazejewicz for the quick review and the useful link!